### ISSUE TITLE
Next/image with imgix loader

### ISSUE THREAD

**Issue by ludwighogstrom** [RyanClementsHax/storybook-addon-next#67]
### Describe the bug

Does not add absolute path to the imgix server for the img src.

### Your minimal, reproducible example

Sorry no repo...

### Steps to reproduce

Use `next/image` component in a story with the following configuration:

next.config
```
  images: {
    loader: "imgix",
    path: "https://my-account.imgix.net",
  },
```

Component with `next/image`
```
<Image src={"7f552320-6810-5a2d-8a47-521b22068ae5"} alt={"Alt"} width={42} height={42} />
```

### Expected behavior

When running Next (`yarn dev`) the img src will automatically be set to `https://my-account.imgix.net/7f552320-6810-5a2d-8a47-521b22068ae5`.

When running Storybook the img src will only be `7f552320-6810-5a2d-8a47-521b22068ae5` right now. This results in a load to localhost and a 404 response.

### How often does this bug happen?

Every time

### Platform

macOS

### storybook-addon-next version

v1.6.2

**Comment by RyanClementsHax** [comment]
Thanks for reporting the issue! This is actually a bug. I'm consistently humbled by how well users can find blind spots in my code lol.

The issue is that I don't load the image config like nextjs does. Right now I ignore it. I'll get to work on this soon.

**Comment by ludwighogstrom** [comment]
Thanks for fast response! Also want to take the opportunity to say thank you for running this package. Just started to use it but really saves a lot of the hassle to make Next work with Storybook.

**Comment by RyanClementsHax** [comment]
Thanks a bunch! That means a lot :) I'm glad it's working out for you.

**Comment by RyanClementsHax** [comment]
This seems to be solved also by solving #70. So check there for updates for now.
